### ISSUE TITLE
complete reference for background & buddy check

### ISSUE THREAD

**Issue by BillMills** [IQuOD/AutoQC#206]
@s-good, could you please provide the complete reference for how EN_std_lev_bkg_and_buddy is performed? The reference linked (http://www.metoffice.gov.uk/hadobs/en3/OQCpaper.pdf) provides a high-level overview, but not all the details.

Context: as you said in #195, there are issues with a negative argument in the logarithm:

```
expArg = -0.5 * np.log(1.0 - rho2) + expArg
```

Looking up a couple lines, we have:

```
rho2 = covar**2 / (errVarA + errVarB)
```

which is a bit suspicious, given that rho2 ought to be unitless judging by its use in the logarithm, and a squared covariance divided by a sum of variances doesn't sound unitless if memory serves (and if I'm guessing the meaning of those variable names correctly). Anyway, this may be a red herring, but I need the full details of the calculation to track down whatever is wrong in the buddy check.

**Comment by s-good** [comment]
This is a very good spot! I've checked back at the original code and it looks like it should be errVarA * errVarB (not +).

**Comment by BillMills** [comment]
resolved in #208
